ui/src/middlewares/redirectToErrorPage.js

```diff
@@ -9,6 +9,9 @@ export default function() {
       // INFO: 'push' and 'replace' methods from 'connected-react-router' 
       // remove error causing url from history. To be able to keep in 
       // and retrieve it from history in error pages we use window API
+      if (error.status === 400) {
+        window.location.assign(`${ERRORS}/500`);
```

Comment:
wouldn't this always redirect to `500`, event if it's an `The syntax of the search query is invalid.`?

Reply:
No, because this middleware only redirects errors which have redirectableError: true property, and in case of invalid query errors I intercept them in search action and set this flag to false for them so they are not redirectable.